app/controllers/main_controller.rb

```diff
@@ -1,4 +1,20 @@
 class MainController < ApplicationController
+  
   def index
+    @card = Card.expired.random    
+  end
+  
+  def check
+    @card = Card.where(original_text: params[:card][:original_text]).take
+    user_text = params[:card][:translated_text]  
+    if !@card.blank?
+      @card.update(review_date: Date.today + 3.days)
+      if @card.translated_text.include? user_text
+        flash[:notice] = 'Правильно'
+      else
+        flash[:error] = 'Не правильно'
+      end
+    end
+    redirect_to action: 'index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.